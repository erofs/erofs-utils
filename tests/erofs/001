#!/bin/sh
# SPDX-License-Identifier: GPL-2.0+
#
# test if unknown algorithm is specificed
#
seq=`basename $0`
seqres=$RESULT_DIR/$(echo $0 | awk '{print $((NF-1))"/"$NF}' FS="/")

# get standard environment, filters and checks
. "${srcdir}/common/rc"

cleanup()
{
	cd /
	rm -f $tmp.*
}

# remove previous $seqres.full before test
rm -f $seqres.full

# real QA test starts here
echo "QA output created by $seq"

if [ -z $SCRATCH_DEV ]; then
	SCRATCH_DEV=$tmp/erofs_$seq.img
	rm -f SCRATCH_DEV
fi

MKFS_OPTIONS="$MKFS_OPTIONS -zunknownalg"
_scratch_mkfs $here/../lib >> $seqres.full 2>&1 && \
	_fail "successfully mkfs with invalid algorithm"

echo Silence is golden
status=0
exit 0
