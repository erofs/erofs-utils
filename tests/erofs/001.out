QA output created by 001
Silence is golden
